README.md

```diff
@@ -1,35 +1,44 @@
-<h1 align="center">
-  <a href="">
-    <img src="/src/assets/happy-thoughts.svg" alt="Project Banner Image">
-  </a>
-</h1>
+# Miko's Happy Thoughts Project ❤️
+The Happy Thoughts project is a React application designed to practice the React state management skills by fetching and posting data to an API. This is a simple API to collect 'happy thoughts' and our own version of Twitter.
 
-# Happy Thoughts
 
-In this week's project, you'll be able to practice your React state skills by fetching and posting data to an API.
+## Feature⛳️
+- Managing React state to handle data fetching and posting.
+- Creating a responsive web application that adapts to different screen sizes.
+- Implementing interactive features like liking thoughts and real-time character counting.
+- Adding animations for a more engaging user experience.
 
-## Getting Started with the Project
+## Credit 🍁
+- The Confetti animation is implemented using the react-confetti library.
+- The useWindowSize hook from react-use is used to enhance the animation.
 
 ### Dependency Installation & Startup Development Server
-
 Once cloned, navigate to the project's root directory and this project uses npm (Node Package Manager) to manage its dependencies.
-
 The command below is a combination of installing dependencies, opening up the project on VS Code and it will run a development server on your terminal.
 
 ```bash
 npm i && code . && npm run dev
+npm i react-confetti 
+npm i react-use
+npm i date-fns
```

Comment:
Interesting to understand that several librairies can be installed to enhance code writing